website/docs/patterns/button-organization/partials/guidelines/alignment.md

```diff
@@ -6,41 +6,42 @@ Alignment refers to how buttons and actions are aligned relative to the containe
 
 When considering internationalization (i18n), **start** and **end** are suitable substitutes for **left** and **right** alignment to account for languages that are read right-to-left ([RTL](https://developer.mozilla.org/en-US/docs/Glossary/RTL)).
 
-![Button alignment options](/assets/patterns/button-organization/alignment-methods.png)
+![](/assets/patterns/button-organization/alignment-methods.png)
 !!!
 
 ### Left alignment
 
-Actions are aligned to the left of the container or page. A common example of this alignment method is in [Modals](/components/modal) and page-level forms.
+In most scenarios, align buttons to the **left side** of a container or page. The primary button should be leftmost, followed by secondary, then tertiary.
 
-![Left alignment example](/assets/patterns/button-organization/using-left-alignment.png)
+- This reinforces natural reading patterns and a conversational "dialog-like" messaging strategy.
+- When combined with a consistent ordering method, the primary button is prioritized in the reading direction.
+- When considering different contexts, opting for a single method helps to create consistency between constrained and unconstrained components and patterns.
+
+A common example of this alignment method is in [Modals](/components/modal) and page-level forms.
 
 !!! Do
 
-In most scenarios, align buttons to the **left side** of a container or page.
+Align buttons to the left side of the container.
 
-- This reinforces natural reading patterns and a conversational "dialog-like" messaging strategy.
-- When combined with a consistent ordering method, the primary button is prioritized in the reading direction.
-- When considering different contexts, opting for a single method helps to create consistency between constrained and unconstrained components and patterns.
+![Modal with buttons in the footer aligned to the left. The reading order goes from left to right, top to bottom.](/assets/patterns/button-organization/alignment-reading-pattern-comparison.png)
 
-![Button alignment reading pattern](/assets/patterns/button-organization/alignment-reading-pattern.png)
 !!!
 
 ### Right alignment
 
-Actions are aligned to the right of the container or page. This alignment method is commonly used for page-level functions or actions that create and manage elements within a page.
+This alignment method is commonly used for page-level functions or actions that create and manage elements within a page. The primary button should be the farthest right, then secondary, then tertiary.
```

Comment:
This now reads a little weird to me, WDYT about this?

```suggestion
In the case of page-level functions or actions that manage elements within the page, align buttons to the right of the page. The primary button should be the farthest right, then secondary, then tertiary.
```

Reply:
Way better, committing.